docs(v2): consolidate and make improvements to static assets docs (#3207)
facebook/docusaurus: website/docs/guides/creating-pages.md

In the same way, a page will be created at `http://localhost:3000/helloMarkdown`.

> Markdown pages allow you to use [markdown features](markdown-features.mdx) in your pages, which may be more convenient than creating React pages. However, markdown pages are less flexible than React pages, because it always uses the theme layout.

Reviewer: ```suggestion
Markdown pages allow you to use [markdown features](markdown-features.mdx) in your pages, which may be more convenient than creating React pages. However, markdown pages are less flexible than React pages, because they always use the theme layout.
```

> Markdown pages allows you to use [markdown features](markdown-features.mdx) in your pages, which may be more convenient than creating React pages. However, markdown pages are less flexible than React pages, because it always uses the theme layout.

Reviewer 2: ```suggestion
Markdown pages allow you to use [markdown features](markdown-features.mdx) in your pages, which may be more convenient than creating React pages. However, markdown pages are less flexible than React pages, because it always uses the theme layout.
```
